src/projects/create-pull-request/create-pull-request.html

```diff
@@ -202,6 +204,7 @@
           this.$.errorToast.show({text: 'Missing pull request description'});
           return;
         }
+        this.loadingPR = true;
```

Comment:
The ajax call to the back-end is in `create-pull-request.html`. Why not bind to the loading property of that element? If creating the PR errors, the loading property of `createPull` will automatically be false.